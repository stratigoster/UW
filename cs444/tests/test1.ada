PACKAGE foo IS
  b : BOOLEAN := true AND THEN NOT 5+3**2-0 OR ELSE false;
--                                 ^^^
-- NOT expects boolean type, given int
END foo;
