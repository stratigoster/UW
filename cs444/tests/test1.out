***ERROR***:2:34:	NOT takes a boolean type

------
Compilation error
------
